hivanreyes/oe-webapp
# Feat/ngpoe make featured expedition component

#### What does this PR do?
Add Featured Expedition Component and its tests.

#### Where should the reviewer start?
1️⃣  components/home/FeaturedExpedition.jsx

#### How should this be manually tested?
👿 You can see the homepage and run the tests with yarn jest --coverage, 100% coverage 🌶 

#### Any background context you want to provide?
Backend should be running.

#### What are the relevant tickets?
N/A.

#### Checklist
